site/_layouts/default.html

```diff
@@ -11,7 +11,7 @@
     {% for lang in unique_languages %}
       <link rel="alternate" hreflang="{{lang}}" href="/{{lang}}{{page.url}}">
     {% endfor %}
-    <title>KeeDrop: securely send passwords | {% t page.title %}</title>
+    <title>KeeDrop: securely send secrets | {% t page.title %}</title>
```

Comment:
translation missing?

Reply:
Heh, I thought about translating it. Didn't think it important to have separate taglines, but I guess it would make sense.

Reply:
SEO man!